lib/builderator/config/attributes.rb

```diff
@@ -102,8 +102,7 @@ def collection(collection_name, &definition)
       include Enumerable
 
       ## Delegate enumerables to underlying storage structure
-      def_delegators :@attributes,
-                     :[], :[]=, :fetch, :store,
+      def_delegators :@attributes, :[], :fetch, :[]=,
```

Comment:
Yeah, it's not currently used at all, and we don't actually want to expose setters on the storage structure. There should already be named setters for any valid attribute. There are a few places where `[]=` is still used, which is why it's still lingering...